seed/tests/test_column_views.py

```diff
@@ -152,6 +152,23 @@ def test_get_all_columns(self):
         # randomly check a column
         self.assertIn(expected, data)
 
+    def test_column_units_added(self):
+
+        responseWithoutUnits = self.client.get(reverse('api:v3:columns-list'), {
+            'organization_id': self.org.id,
+            'display_units': 'false'
+        })
+        columnWithoutUnits = next((x for x in json.loads(responseWithoutUnits.content)['columns'] if x['id'] == 61), None)
```

Comment:
Sorry - I definitely led you down this path with my last comment. I was envisioning using something like, for the column you chose,  `x['column_name'] == 'site_eui_modeled'`. The ID won't be consistent because of all the tear downs that happen between tests when this test is run with the full test suite.

To clarify my previous "well-defined" comment: when a new Organization is created, it always comes with several Column objects that have the `units` field populated. These Column objects have `column_name` values of "site_eui", "site_eui_modeled", "source_eui" among a few others.